lib/util/isNextLineNotEmpty.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * This file is part of the Sententiaregum project.
+ *
+ * (c) Maximilian Bosch <maximilian.bosch.27@gmail.com>
+ * (c) Ben Bieler <benjaminbieler2014@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+module.exports = function (nextLine) {
+  return typeof nextLine === 'undefined' ? false : nextLine.trim().length !== 0;
+};
```

Comment:
this module could be removed as this is a one-line function used at only one part of the implementation